app/models/assignment_survey_response_map.rb

```
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 class AssignmentSurveyResponseMap < SurveyResponseMap
-    include ResponseMapSubclassTitles
+    include ExpertizaConstants::ResponseMapTitles
```

RuboCop (1.86.1)

[convention] Use 2 (not 4) spaces for indentation.

(Layout/IndentationWidth)